### PR TITLE
initial version training

### PR DESCRIPTION
**Pull request by eaidova**
- [x] requirements 

- [x] data preprocessing

- [x]  training code

- [x] configs

- [x]  evaluation on test set

- [x]  tensorboard logging

- [x] docs

### CODE REVIEW

**Comment by eaidova** [comment]
@grib0ed0v please take a look
